helpers/test/test_lib_tasks_docker_release.py

```diff
@@ -353,7 +384,403 @@ def test_docker_build_prod_image_with_candidate_user_tag(self) -> None:
+            self.mock_ctx,
+            self.test_version,
+            target_registry=target_registry,
+        )
+        exp = r"""
+        docker buildx imagetools create -t causify/test-image:prod-1.0.0 test.ecr.path/test-image:prod-1.0.0
+        docker buildx imagetools create -t causify/test-image:prod test.ecr.path/test-image:prod-1.0.0
+        """
+        self._check_docker_command_output(exp, self.mock_run.call_args_list)
+
+
+# #############################################################################
+# Test_docker_tag_push_multi_build_local_image_as_dev1
+# #############################################################################
+
+
+class Test_docker_tag_push_multi_build_local_image_as_dev1(_DockerFlowTestHelper):
+    """
+    Test tagging and pushing a multi-arch local Docker image as dev.
+    """
+
+    def test_aws_ecr1(self) -> None:
+        """
+        Test pushing to AWS ECR.
+
+        This test checks:
+        - Multi-arch image tagging
+        - AWS ECR target registry
+        - Dev image versioning
+        - Default and versioned tagging
+        """
+        # Call tested function.
+        target_registry = "aws_ecr.ck"
+        hltadore.docker_tag_push_multi_build_local_image_as_dev(
+            self.mock_ctx,
+            self.test_version,
+            target_registry=target_registry,
+        )
+        exp = r"""
+        docker buildx imagetools create -t test.ecr.path/test-image:dev-1.0.0 test.ecr.path/test-image:local-$USER_NAME-1.0.0
+        docker buildx imagetools create -t test.ecr.path/test-image:dev test.ecr.path/test-image:local-$USER_NAME-1.0.0
+        """
+        self._check_docker_command_output(exp, self.mock_run.call_args_list)
+
+    def test_dockerhub1(self) -> None:
+        """
+        Test pushing to DockerHub from AWS ECR.
+
+        This test checks:
+        - Multi-arch image tagging
+        - DockerHub registry (differs from AWS ECR test)
+        - Version and latest tagging
+        - Cross-registry image copying
+        """
+        # Call tested function.
+        target_registry = "dockerhub.causify"
+        hltadore.docker_tag_push_multi_build_local_image_as_dev(
+            self.mock_ctx,
+            self.test_version,
+            target_registry=target_registry,
+        )
+        exp = r"""
+        docker buildx imagetools create -t causify/test-image:dev-1.0.0 test.ecr.path/test-image:local-$USER_NAME-1.0.0
+        docker buildx imagetools create -t causify/test-image:dev test.ecr.path/test-image:local-$USER_NAME-1.0.0
+        """
+        self._check_docker_command_output(exp, self.mock_run.call_args_list)
+
+
+# #############################################################################
+# Test_docker_release_dev_image1
+# #############################################################################
+
+
+class Test_docker_release_dev_image1(_DockerFlowTestHelper):
+
+    def test_docker_release_dev_image(self) -> None:
```

Comment:
Let's simplify the method name according to GP comment: 
https://github.com/causify-ai/helpers/pull/707#discussion_r2087191455

Reply:
Same everywhere.